### DISCUSSION TITLE
Wakatime plugin

### DISCUSSION THREAD

**Discussion by jaredgizersky** [zed-industries/zed#8147] [Feature Requests]
Is there any (even rough) implementations for a plugin? I want to start using Zed more but the fact that I can't track my coding is a big negative. Building out a plugin for wakatime would be a project I'm interested in, but I am wondering if there is even the infrastructure to achieve this.

**Comment by Moshyfawn** [comment]
There's an issue #4738 tracking WakaTime support.

As for plugins in Zed, it's not there yet, but they [have recently released](https://arc.net/l/quote/qrtowpbc) initial support for language and theme extensions.
